### DISCUSSION TITLE
How can I modify the parameters of a single stage and run that stage repeatedly？

### DISCUSSION THREAD

**Discussion by huang-kaixin** [The-OpenROAD-Project/OpenROAD-flow-scripts#1674] [Q&A] [answered]
I have already run a floorplan (make floorplan) with the default CORE UNITION set to 55 once. Then, I tried to modify CORE UNITION to 40 and run floorplan again to observe the changes in the floorplan stage. However, whether I modify CORE UNITION to 40 in config.mk and run make floorplan, or run make floorplan CORE UNITION=40 directly, it prompted “make: Nothing to be done for 'floorplan' ”. I want to know how to modify the parameters of an individual stage and make it run again and overwrite the original run? Thank you for your reply.

**Comment by maliberty** [comment, marked as answer]
You can do 'make clean_floorplan' to remove the prior floorplan result
